src/MessagePack.SourceGenerator/CodeAnalysis/CustomFormatterRegisterInfo.cs

```diff
@@ -13,4 +13,6 @@ public override string GetFormatterInstanceForResolver()
             ? base.GetFormatterInstanceForResolver()
             : $"{this.GetFormatterNameForResolver()}.{this.CustomFormatter.InstanceProvidingMember}";
     }
+
+    public override int GetHashCode() => base.GetHashCode();
```

Comment:
Sorry for the late reply. I thought `GetHashCode` in base class is sufficent to hash the instances and tried to avoid use every fields to hash, so I made child classes to use `GetHashCode` in base class.